package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./server/server.js",
   "scripts": {
     "test": "./node_modules/.bin/jest",
-    "test:watch": "./node_modules/.bin/jest -w",
+    "test:watch": "./node_modules/.bin/jest --watch",
```

Comment:
does this make jest actively watch your test files for changes?

Reply:
Yeah, I thought the flag was -w but it is actually --watch.  jest will auto-re-run when it notices changes in any of your files.